Server/Server-Leaderstats.lua

```diff
@@ -77,4 +77,22 @@ return function(Vargs)
 			end
 		end
 	}
+
+	Commands.RemoveAllStats = {
+		Prefix = Settings.Prefix;
+		Commands = {"removestats", "rstats"};
+		Description = "Removes all the stats";
+		AdminLevel = "Moderators";
+		Function = function(plr)
+			local thestats = plr.leaderstats:GetChildren()
+			
+			if thestats then
+				plr.leaderstats:ClearAllChildren()
+				wait(1)
```

Comment:
Wait is deprecated
https://create.roblox.com/docs/reference/engine/libraries/task

Reply:
I tested it and it worked.
Ill check again later

Reply:
Just checked and its not deprecated

Reply:
`task.wait` is more accurate

Reply:
alr